clients/python/src/raystack/stencil.py

```diff
@@ -0,0 +1,30 @@
+from .store import Store
```

Comment:
can we use absolute path instead of relative path ?

Reply:
done